README.md

```diff
@@ -1,10 +1,47 @@
 # FairCMakeModules
+
 CMake Modules developed in the context of various FAIR projects
 
+## Installation
+
+```
+git clone https://github.com/FairRootGroup/FairCMakeModules
+cmake -S FairCMakeModules -B FairCMakeModules_build -DCMAKE_INSTALL_PREFIX=FairCMakeModules_install
+cmake --build FairCMakeModules_build --target install
+```
+
+## Usage
+
+In your project, discover this package with [`find_package`](https://cmake.org/cmake/help/latest/command/find_package.html#full-signature-and-config-mode):
+
+```
+find_package(FairCMakeModules)
+```
+
+Then [`include`](https://cmake.org/cmake/help/latest/command/include.html) any module you would like to use
+
+```
+include(<modulename>)
+```
+
+Available modules are:
+
+* [FairFindPackage2](src/modules/FairFindPackage2.cmake)
```

Comment:
What about just linking to `src/modules/` and be done?

Reply:
Hm, I would like to have this list on the landing page. Otherwise, people have to read the generic boring instructions, and find the links, click it and only then really get to the interesting part. I think it is worth the additional maintenance of the README :)

Reply:
Agreed.